perf/auto-qual-test/deploy_latest_daily.sh

```diff
@@ -0,0 +1,35 @@
+git clone https://github.com/istio/tools
+cd tools/perf/istio-install
+helm init --client-only
+TARGET_VERSION=$1
+LATEST_BUILD=$(curl -L https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/release-$TARGET_VERSION-latest.txt)
+HELMREPO_URL=https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/$LATEST_BUILD/charts/ RELEASE_URL=https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/$LATEST_BUILD/istio-$LATEST_BUILD-linux.tar.gz DNS_DOMAIN=v11.qualistio.org ./setup_istio.sh $TARGET_VERSION
+
+NAMESPACES=$(kubectl get namespace -l istio-injection=enabled --no-headers -o custom-columns=":metadata.name")
+
+# trigger redeploy on services to get new sidecars
+# based on the "patch deployment" strategy in this comment:
+# https://github.com/kubernetes/kubernetes/issues/13488#issuecomment-372532659
+# requires jq
+
+# $1 is a valid namespace
+function refresh-all-pods() {
```

Comment:
Can we reused `/bin/redeploy.sh`?

Reply:
`/bin/redeploy.sh ALL`

Reply:
I didn't know that existed!  Why don't our docs point to this?  This is great!  Will update.